Lecture3/Task1/Task1L3.java

```diff
@@ -0,0 +1,246 @@
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Task1L3 {
+
+    private static List<Voucher> listOfVouchers = new ArrayList<>();
+    private static List<Voucher> listOfSelection = new ArrayList<>();
+    private static Map<Integer, Voucher> mapOfSelection = new TreeMap<>();
```

Comment:
why do we need to store selected vouchers as map?

Reply:
fixed